pom.xml

```diff
@@ -35,40 +35,38 @@
         <revision>6.0.0</revision> <!-- CHANGE THIS to adjust project version-->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
-        <!-- Use 1.0.6 for fasterxml v 2.12.x (good for databricks runtime 2.10.x) -->
-        <azure-bom-version>1.2.18</azure-bom-version>
+        <azure-bom-version>1.2.22</azure-bom-version>
 
         <!-- Versions below are for several dependencies we're using in the data & ingest modules -->
         <!-- Ideally, versions below should align with latest databricks runtime dependency versions -->
         <!-- Compile dependencies -->
```

Comment:
One of the things we have to fix is the dependencies for SL4J, Reactor (removed in this version), Jackson (discussed below) and Netty (not explcitly added should be okay).

Adding custom versions causes issues downstream for apps using SDK to build on. 

Ref : https://learn.microsoft.com/en-us/azure/developer/java/sdk/troubleshooting-dependency-version-conflict